fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccount.java

```diff
@@ -290,6 +292,12 @@
     @Column(name = "min_required_balance", scale = 6, precision = 19, nullable = true)
     private BigDecimal minRequiredBalance;
 
+    @Column(name = "is_lien_allowed")
+    private boolean lienAllowed;
```

Comment:
Use Boolean to make it null safe

Reply:
No need to. If `lienAllowed` is a binary switch then let's stick with primitive boolean. Less headache when dealing with it's value. Just make sure the database column is not nullable.